Casamari/MNC010.xml

```diff
@@ -122,77 +215,163 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                           (e g. <locus target="#34v #35r"/>). </desc>
+                        <seg type="rubrication">Two initial lines, and occassionally two more lines
+                           further down, in the incipt of all texts; first word of each stanza of
+                              <ref target="#ms_i1 #ms_i3 #ms_i8"/>; holy names; the name of the
+                           apparent owner <persName ref="PRS13232EhetaMar"/>; elements of the
+                           punctuation marks, text dividers and of numerals.</seg>
+                        <list type="abbreviations">
+                           <item>
+                              <abbr>መ</abbr> for <expan>መለኮት፡ </expan> (<ref target="#ms_i8"/>)
+                           </item>
+                        </list>
+                     </handNote>
+
+                     <handNote xml:id="h2" script="Ethiopic" corresp="#a1">
+                        <locus from="2r" to="2v"/>
+                        <seg type="ink">Black</seg>
+                        <desc>Mediocre. Broad script with hairlines.</desc>
                      </handNote>
                   </handDesc>
+
                   <additions>
                      <list>
-                        <item xml:id="a1">
-                           <desc type="Doxology">Prayer of blessing</desc>
+                        <item xml:id="a1" corresp="#h2">
                            <locus from="2r" to="2v"/>
-                           <desc>The note begins as follows: 
-                              <q xml:lang="gez">
-                                 <lb n="1"/>ብፁዓን፡ እለ፡ የኀሡ፡ ስምዖ። 
-                                 <lb n="2"/>ወበኵሉ፡ ልቦሙ፡ የኃሥዎ። ኦሐሩ። 
-                                 <lb n="3"/>እለሰ፡ ይገብሩ፡ ዓመፃ፡ ፍኖተ፡ ዚአሁ፡ 
-                                 <lb n="4"/>እንተ፡ አዘዝከ፡ የዕቀቡ፡ ትእዛዘከ፡
-                              </q>
+                           <desc type="GuestText">
+                              <title ref="LIT2701Dawit">Excerpt from the Psalter</title>
                            </desc>
-                        </item> 
-                        </list>
+                           <note>The excerpt comprises <ref type="work"
+                                 corresp="LIT2701Dawit.118.2-14">Ps 118,2–4</ref>.</note>
+                           <q xml:lang="gez">
+                              <locus target="#2r"/>
+                              <seg part="I">
+                                 <lb n="1"/>ብፁዓን፡ እለ፡ የኀሡ፡ ስምዖ። <lb n="2"/>ወበኵሉ፡ ልቦሙ፡ የኃሥዎ። <lb
+                                    n="3"/>እለሰ፡ ይገብሩ፡ ዓመፃ፡ ፍኖተ፡ ዚአሁ፡ ኦሖሩ።<lb n="4"/>እንተ፡ አዘዝከ፡ ይዕቀቡ፡
+                                 ትእዛዘከ፡ ፈድፋደ።</seg>
+                           </q>
+                        </item>
+
+                        <item xml:id="e1">
+                           <desc> Scribal corrections occur interlineally (e.g. <locus
+                                 target="#4r #5v #7v #8r"/>), occasionally over erasure (e.g. <locus
```

Comment:
I'd stick to a maximum of three examples after e.g. (also below on erroneous characters encircled).
Also, in the other PR Jacopo has said "over erasures" sounds better than "over erasure". In case we approve the formula "over erasures", we should correct it also here.